### PR TITLE
update fusion to version 1.1.1

### PR DESCRIPTION
**Pull request by jirgn**
* introduces afx support
* fix highlights and queries

### CODE REVIEW

File: lockfile.json [hunk 1]
```diff
@@ -246,7 +249,7 @@
     "revision": "111b07762e86efab9a918b7c721f720c37e76b0a"
   },
   "verilog": {
-    "revision": "8f6b1f357d1231c420404b5f7a368a73c25adfa2"
+    "revision": "6fae7414fa854b5052bee9111b200e9137797f3d"
```
Comment (theHamsta) [inline lockfile.json:252]: please do not update verilog! It's incompatible.

File: lockfile.json [hunk 2]
```diff
@@ -158,6 +158,9 @@
   "nix": {
     "revision": "6d6aaa50793b8265b6a8b6628577a0083d3b923d"
   },
+  "norg": {
+    "revision": "665736e400cfd52ae92ead244ca9f5d44db98151"
+  },
```
Comment (theHamsta) [inline lockfile.json:163]: unrelated

File: lockfile.json (hunk 1 above)
Comment (jirgn) [reply lockfile.json]: I should have know - did this the last time.
I try to put my glasses on next time ;)
Should be fixed

File: queries/fusion/highlights.scm [hunk 3]
```diff
@@ -1,7 +1,24 @@
...
+; identifiers afx
+(afx_opening_element
+  (afx_identifier) @tag)
+(afx_closing_element
+  (afx_identifier) @tag)
+(afx_element_self_closing
+  (afx_identifier) @tag)
+
+(afx_attribute
+  (afx_property_identifier) @attribute)
```
Comment (theHamsta) [inline queries/fusion/highlights.scm:13]: This is probably `@tag.attribute` https://github.com/theHamsta/nvim-treesitter/blob/1111827b36324bc4e80b6d493e61a85aeca592ef/CONTRIBUTING.md#L187
`@attribute` is more like C++-attribute (an annotation).

**Review by theHamsta — APPROVED** [review]

**Comment by theHamsta** [comment]
@jirgn could you fix the conflict on rebase?

**Comment by stsewd** [comment]
squashing works!
